f/pabstMarkers/fn_getGroupMarkerStyle.sqf

```diff
@@ -1,175 +1,176 @@
+private ["_red","_green","_blue","_yellow","_orange","_black","_pink","_white","_fireteam","_unknown","_hq","_support","_supportAT","_recon","_mortar","_maint","_mech","_armor","_air","_plane","_artillery","_med","_size","_input","_texture","_color"];
```

Comment:
Same as the other file, Mike's planning on making some changes to this file so I'm going to redo it anyway.